pkgs/development/libraries/cpp-utilities/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  name = "cpp-utilities-${version}";
+  version = "4.17.1";
+
+  src = fetchFromGitHub {
+    owner = "Martchus";
+    repo = "cpp-utilities";
+    rev = "v${version}";
+    sha256 = "12088cwg3jbqipmbn4843w1cgxi1q6vwx47gy042rkfvbk6azhxl";
+  };
+
+  outputs = [ "out" "dev" ];
+  doCheck = true;
+
+  nativeBuildInputs = [ cmake ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/Martchus/cpp-utilities;
```

```suggestion
    homepage = "https://github.com/Martchus/cpp-utilities";
```
Hint: We're probably going to deprecate unquotes URL syntax when RFC 45 is approved.